### ISSUE TITLE
"..." shows on Android Navigation bar

### ISSUE THREAD

**Issue by wzfcooler** [AppGyver/steroids#953]
It is reproducible on /myProject example as well as kitchensink example.
Navigatioin bar show Text correctly on iOS emulator, my iPhone 6, and other Android phones I've tested. But it shows only three dots "..." on one of the Android device.
It doesn't help when I change css text size or weight.

Some details about the particular device that fails on Navigation bar:
A Samsung Note 3 NEO phone with Android version 4.4.2 installed

![screenshot_2015-05-29-19-50-33](https://cloud.githubusercontent.com/assets/5104336/7889152/fb74af56-063d-11e5-8bde-6da511b93e10.png)

**Comment by harsaharsa** [comment]
What is the code initializing the `<super-navbar>` component? It could be a bug that doesn't strip extra whitespace on Android that might have resurfaced. Can you see if changing it like `<super-navbar>My Title</super-navbar>` (i.e. no line changes) has an effect?

**Comment by wzfcooler** [comment]
Precisely, it is because of empty space, after change as you suggested, it displays all right. Thanks!
